### DISCUSSION TITLE
trim works in the default template but when copied in, it gives an error

### DISCUSSION THREAD

**Discussion by abesmith1** [hadynz/obsidian-kindle-plugin#204] [Q&A]
Greetings. I love this plugin, but one thing I am having a problem with is modifying the default book template. If  {% trim %} ...{% endtrim%} is included in my template, the box turns red and doesn't load - even though {% trim %} ...{% endtrim%} is in the default template.

Anyone else have any advice on how to navigate it? I guess it isn't terrible without trim - just a few extra lines in the rendered file.

**Comment by nschutz** [comment]
also running into this issue, trying to find an example of anyone's template in forums but I haven't found any yet. I think for me it's anything that uses `{% ` doesn't allow me to save my changes. Not sure if this is a dependency I don't have or something

**Comment by Titus-314** [comment]
Yup, finding the same. I'd like to keep that template and add one or two items only.

To replicate:
1) Find the template file in Github and copy the template text
2) Paste into the "File Template" section in the community plugins
3) See red box around the editor and when you navigate away it doesn't save.

Making minor amendments doesn't change the red box error.
![image](https://user-images.githubusercontent.com/32364522/200124078-6085ac4e-8607-4129-8c17-e06c3434bcda.png)
